include/faker-cxx/Helper.h

```diff
@@ -55,6 +59,173 @@ class Helper
         return data;
     }
 
+    /**
+     * @brief Returns the given string parsed symbol by symbol and replaced the placeholders with digits ("0" - "9").
+     * "!" will be replaced by digits >=2 ("2" - "9").
+     *
+     * @param str The template to parse string.
+     *
+     * @param symbol The symbol to replace with digits. Defaults to '#'.
+     *
+     * @return The string replaced symbols with digits.
+     *
+     * @code
+     * Helper::replaceSymbolWithNumber() // ""
+     * Helper::replaceSymbolWithNumber("#####") // "04812"
+     * Helper::replaceSymbolWithNumber("!####") // "27378"
+     * Helper::replaceSymbolWithNumber("Your pin is: !####") // "29841"
+     * @endcode
+     */
+    static std::string replaceSymbolWithNumber(std::string str, const char & symbol = '#')
```

Comment:
please move implementations in source file

Reply:
Done